docs/content/installation/install_compatibility_requirements.md

```diff
@@ -50,7 +50,7 @@ Complete these steps to prepare your environment for installing the CSI (Contain
 
 2. Configure your storage system host attachment, per worker node.
 
-    **Note:** IBM® block storage CSI driver 1.11.0 introduces dynamic host definition. For more information and installation instructions, see [Installing the host definer](install_hostdefiner.md). If this feature is not installed, the nodes are not dynamically defined on the storage system and they must be defined manually.
+    **Note:** IBM® block storage CSI driver 1.11.1 introduces dynamic host definition. For more information and installation instructions, see [Installing the host definer](install_hostdefiner.md). If this feature is not installed, the nodes are not dynamically defined on the storage system and they must be defined manually.
```

Comment:
are we okay with saying that?
since it's not 1.11.1 that introduced host definition feature

Reply:
fixed

Reply:
I don't see that

Reply:
saw it, thanks